### ISSUE TITLE
Plane shape crashes (v21)

### ISSUE THREAD

**Issue by HexaField** [enable3d/enable3d#75]
using `enable3d@0.21.0` and `three r123`

```
Uncaught (in promise) TypeError: Cannot read property 'length' of undefined
    at Object.createTriMeshShape (three-to-ammo.js:457)
    at AmmoPhysics.createCollisionShape (physics.js:628)
    at AmmoPhysics.addExisting (physics.js:704)
    at Object.existing (physics.js:455)
    at threeScene.ts:35
```

To replicate
```js
const floor = new THREE.Group()
floor.add(new THREE.Mesh(new THREE.PlaneBufferGeometry(), new THREE.MeshBasicMaterial()));
this.scene.add(floor);
this.physics.add.existing(floor, { shape:'plane' })
```

It seems that https://github.com/enable3d/enable3d/blob/6d37cf8543bde2e90111e6f774988b9738419bf9/packages/ammoPhysics/src/physics.ts#L710 is the culprit since in this specific case, since the if statement returns false, d remains empty but then is passed anyway, so the vertices/indices/matrices/matrixWorld are undefined. I'm curious as to the purpose of the if statement.

Thanks.

**Comment by yandeu** [comment]
We only extract the data if you want to make a complex shape (using three-to-ammo). But since there is no plane shape, it also uses a complex shape (convexMesh). Unfortunately, the plane is at index 0 in the list of complex shapes (complexShapes). I have fixed it (https://github.com/enable3d/enable3d/commit/77b214136956355ed777d4c45f69129b41b9a111).

---

For now, please fall back to a convexMesh shape, since it is the same.

```ts
this.physics.add.existing(floor, { shape: 'convexMesh' })
```

**Comment by HexaField** [comment]
Thank you! Confirmed this works.

**Comment by yandeu** [comment]
Is fixed in version `0.22.0-dev.0` and above.
